docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -54,7 +54,8 @@ Running dual-region setups requires the users to be able to detect any regional
   - In that guide, we're showcasing Kubernetes dual-region installation, based on the following tools:
     - [Helm (3.x)](https://helm.sh/docs/intro/install/) for installing and upgrading the [Camunda Helm chart](https://github.com/camunda/camunda-platform-helm).
     - [Kubectl (1.30.x)](https://kubernetes.io/docs/tasks/tools/#kubectl) to interact with the Kubernetes cluster.
-- [zbctl](./../../../apis-tools/cli-client/index.md) to interact with the Zeebe cluster.
+- (deprecated) [zbctl](./../../../apis-tools/cli-client/index.md) to interact with the Zeebe cluster.
```

Comment:
Quick comment here: Internally, the discussion has called this "deprecation," but externally, we are moving this to community support; see https://github.com/camunda/camunda-docs/pull/4228. I'm leaning toward removing this line and just promoting cURL.

Reply:
Thanks for the link! I wasn't sure if the blog went live with the deprecation terminology - still catching up from my recent FTO. I will resolve this.